examples/p-chain/etna/utils/etna-context.ts

```diff
@@ -9,5 +8,24 @@ export const getEtnaContextFromURI = async (
 ): Promise<Context.Context> => {
   const context = await Context.getContextFromURI(uri);
 
-  return context;
+  const info = new Info(uri);
+
+  const { etnaTime } = await info.getUpgradesInfo();
+
+  const etnaDateTime = new Date(etnaTime);
+  const now = new Date();
+
+  if (etnaDateTime < now) {
+    return context;
+  }
+
+  // If Etna is not enabled, we need to override the minPrice of 1n that is returned.
+  // This is because the minPrice of 1n is not enough to calculate a fee that exceeds the current static fees.
+  return {
+    ...context,
+    platformFeeConfig: {
+      ...context.platformFeeConfig,
+      minPrice: 10_000n,
```

Comment:
I'm confused by this. Why would we be using the etna builders if etna is not enabled? If we are not using the etna builders than the we do not need the platformFeeConfig at all?

Reply:
This is just to keep these etna examples working right now with the Etna devnet (which still doesn't have Etna upgrade enabled).

If you currently send a `platform.getFeeState` RPC call to the Etna devnet, you will get back a response like:
```json
{
  "jsonrpc": "2.0",
  "result": {
    "capacity": 0,
    "excess": 0,
    "price": 1,
    "timestamp": "2024-10-08T04:23:04Z"
  },
  "id": 1
}
```

As you can see, the `price` is currently set to `1`, and that is not a high enough value to construct a fee higher than the static fees, so the example transactions will fail. So that's why we manually override the `minPrice` (which is also currently `1`) so that we use `10_000n` currently when doing our fee calculations for transactions.